K2Bridge/RewriteRules/RewriteSearchRule.cs

```diff
@@ -72,5 +41,11 @@ private string GetIndexNameFromPath(PathString pathString)
             var segments = pathString.ToString().Split('/', System.StringSplitOptions.RemoveEmptyEntries);
             return segments[0];
         }
+
+        private string GetIndexNameFromResolvePath(PathString pathString)
```

Comment:
Can you extract the index search pattern implicitly by decorating the controller method instead of using this method explicitly?

Reply:
Thanks, Tamir, done.